tools/SourceKit/lib/SwiftLang/SwiftSourceDocInfo.cpp

```diff
@@ -1365,36 +1307,30 @@ static void resolveCursor(
           Length = 0;
       }
 
-      // Retrive relevant actions on the code under selection.
-      std::vector<RefactoringInfo> AvailableRefactorings;
+      // Retrieve relevant actions on the code under selection.
+      llvm::SmallVector<RefactoringInfo, 32> Actions;
```

Comment:
32 seems like a lot and doesn't make much sense when you're only using 16 for `Kinds` when Kinds are 1 byte each while this `Actions` is three words or so.

Reply:
There's 14 cursor refactorings and 11 range refactorings. `Kinds` is only ever range refactorings, hence the 16. `Actions` can be both cursor and refactorings when it's a expr/stmt start, hence the 32. Though in saying that, I used 16 for ranges + cursor inside `passCursorInfoForDecl`.

I'll change to 16 since it is unlikely every refactoring will apply.